.github/workflows/build.yml

```diff
@@ -44,3 +44,4 @@ jobs:
           EXTRA_TWISTER_FLAGS="--short-build-path -O/tmp/twister-out"
         fi
         west twister --test sample.basic.helloworld -p siwx917_rb4338a -v --inline-logs $EXTRA_TWISTER_FLAGS
+        west twister -T ../zephyr/samples -s sample.net.wifi -p siwx917_rb4338a -v --inline-logs -K $EXTRA_TWISTER_FLAGS
```

Comment:
I don't think you need the `-T ../zephyr/samples` part (the line above doesn't have it either), and let's be consistent with using either `--test` or `-s` but not mix of them.